### PR TITLE
Show recipe

### PR DESCRIPTION
**Pull request by easbell**
-When smoothie is clicked, individual smoothie along with recipe will render to page.

-Recipe component added to show alongside selected smoothie.

-Conditional rendering utilizing state to toggle between showing smoothie and showing entire list.

### CODE REVIEW

**Comment by easbell** [comment]
@thatPamIAm @brittanystoroz @robbiejaeger
Second PR for Blend It.

File: src/Recipe.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-export class Recipe extends Component {
-  render() {
-    return (
-      <div>
-        {/* Individual Recipe */}
-      </div>
-    )
-  }
+export default function Recipe(props) {
+  console.log(props)
```
Comment (jessicalyn) [inline src/Recipe.js]: Remove console log

File: src/Smoothie.js [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+    this.state = {
+      isSelected: false
+    }
   }
 
+  expandRecipe = () => {
+    const {isSelected} = this.state
+    this.setState({isSelected: !isSelected})
+    this.props.showRecipe(this.props)
+    console.log(this.props.recipe)
```
Comment (jessicalyn) [inline src/Smoothie.js]: Remove console log

File: src/Smoothie.js [hunk 3]
```diff
@@ -1,15 +1,37 @@
...
     super(props)
+    this.state = {
+      isSelected: false
+    }
   }
 
+  expandRecipe = () => {
+    const {isSelected} = this.state
+    this.setState({isSelected: !isSelected})
+    this.props.showRecipe(this.props)
```
Comment (jessicalyn) [inline src/Smoothie.js]: Should we destructure this.props as well as this.state to stay consistent?

File: src/Recipe.js (hunk 1 above)
Comment (Rosebud303) [inline src/Recipe.js]: Take out console.log() in fina; push

File: src/Smoothie.js [hunk 4]
```diff
@@ -1,15 +1,37 @@
...
+  
   render = () => {
+    const {isSelected} = this.state
     return (
       <div>
-        {this.props.name}
-        <img src={this.props.img}/>
+        {!isSelected && 
+          <div className='smoothie' onClick={this.expandRecipe}>
+            <h3>{this.props.name}</h3>
```
Comment (jessicalyn) [inline src/Smoothie.js]: Destructure this.props to stay consistent

File: src/Recipe.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
+  return (
+    <div>
+      {
+        props.recipe.map(steps => {
+          return (
+            <p>{steps}</p>
```
Comment (Rosebud303) [inline src/Recipe.js]: may want an <li> here instead of a <p> since the recipe shows as a list

File: src/Smoothie.test.js [hunk 6]
```diff
@@ -30,18 +30,28 @@ source: "https://www.modernhoney.com/6-healthy-superfood-smoothies/",
...
 
+const showRecipeMock = jest.fn();
 
 describe('Smoothie', () => {
   const wrapper = shallow(
-    <Smoothie 
-      name = {mockSmoothie.name}
-      key = {mockSmoothie.id}
-      img = {mockSmoothie.img}
+    <Smoothie
```
Comment (jessicalyn) [inline src/Smoothie.test.js]: Good job updating the tests as we go!

File: src/Smoothie.js (hunk 2 above)
Comment (Rosebud303) [inline src/Smoothie.js]: take out console.log() for final push

File: src/SmoothieContainer.js [hunk 7]
```diff
@@ -1,20 +1,57 @@
...
+    }
+  }
+
+  showRecipe = (smoothie) => {
+    const {showRecipe, chosenSmoothie} = this.state;
+    this.setState({showRecipe: !showRecipe, chosenSmoothie: smoothie.id})
+  }
+
+  render = () => {
+    const {showRecipe} = this.state;
```
Comment (jessicalyn) [inline src/SmoothieContainer.js]: Combine const for destructuring

File: src/Smoothie.js (hunk 4 above)
Comment (easbell) [reply src/Smoothie.js]: Let's wait and do this locally, that way we can go through all files and make sure consistency is there.

File: src/Recipe.js [hunk 8]
```diff
@@ -1,13 +1,16 @@
...
-      </div>
-    )
-  }
+export default function Recipe(props) {
+  return (
+    <div>
+      {
+        props.recipe.map(steps => {
+          return (
+            <li>{steps}</li>
```
Comment (brittanystoroz) [inline src/Recipe.js]: `li` tags should have an outter `ul` -- not a `div` ;)

File: src/Smoothie.js [hunk 9]
```diff
@@ -1,15 +1,37 @@
...
   constructor(props) {
     super(props)
+    this.state = {
+      isSelected: false
+    }
   }
 
+  expandRecipe = () => {
+    const {isSelected} = this.state
+    const {showRecipe} = this.props
```
Comment (brittanystoroz) [inline src/Smoothie.js]: Nice use of ES6 destructuring, you'll see this all over the place in React apps.

**Pull request merged**